.travis.yml

```diff
@@ -8,5 +8,5 @@ before_script:
 
 script:
   - make
-  - cd ../../../tests
-  - ./test_networks.sh
+#  - cd ../../../tests
```

Comment:
this disables the tests. is that intended?

Reply:
Yes this was intentional. This pull request deprecates the old version of the `ouputapi` and python wrapper written by @bemcdonnell. The test script written by @samhatchett was also deprecated in this branch to make way for `nrtest` and  `epanet-nrtestsuite`. That was the thought anyway.

Reply:
ok, so i guess i'm a little lost - how are tests going to be run now via Travis?

Reply:
Discussion over at #97 led me to believe we were contemplating using `nrtest` to implement regression testing through CI. It was a while ago :blush:

Reply:
I see your point we could have just modified the contents of `test_networks.sh` as opposed to deleting it. Guess I got a little ahead of myself.

Reply:
Just wanted to make sure that running the tests would still be part of the CI, because at this commit, Travis only builds and nothing else.